insights/client/apps/compliance/__init__.py

```diff
@@ -93,7 +93,7 @@ def download_tailoring_file(self, profile):
 
     def get_profiles(self, search):
         response = self.conn.session.get("https://{0}/compliance/profiles".format(self.config.base_url),
-                                         params={'search': search})
+                                         params={'search': search, 'relationships': False})
```

Comment:
I've tested this locally with the backend change, and we need to change it to a string:

```suggestion
                                         params={'search': search, 'relationships': 'false'})
```

With the current implementation, we get a string param in ruby instead of a boolean: `{"relationships"=>"False"}`

Reply:
Oh okay, gotcha I can make this update now.